src/main/java/pl/lodz/p/edu/flanki/entities/Event.java

```diff
@@ -0,0 +1,34 @@
+package pl.lodz.p.edu.flanki.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@Entity
+@Table(name = "EVENTS")
+@NoArgsConstructor
+@AllArgsConstructor
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "date")
+    private LocalDateTime date;
+
+    @Column(name = "location")
+    private String location;
+
+    @Column(name = "description")
+    private String description;
```

Comment:
Here I believe should be list of ownerIds and participantsIds which should link to userIds